R-package/DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: lightgbm
 Type: Package
 Title: Light Gradient Boosting Machine
-Version: 2.3.2
-Date: 2019-11-26
+Version: 3.0.0-rc.1
```

Comment:
It seems R doesn't support `rc` in version number. 
I am going to replace it with `3.0.0.1` in R.